lib/content-services/src/lib/directives/check-allowable-operation.directive.spec.ts

```diff
@@ -16,12 +16,11 @@
  */
 
 import { ChangeDetectorRef, Component, ElementRef, SimpleChange } from '@angular/core';
-import { ContentService } from '../services/content.service';
-import { CheckAllowableOperationDirective, NodeAllowableOperationSubject } from './check-allowable-operation.directive';
-import { setupTestBed } from '../testing/setup-test-bed';
+import { ContentService, CoreTestingModule, setupTestBed } from '@alfresco/adf-core';
```

Comment:
ContentService from core sounds wrong... if is content should be under content package

Reply:
You're right, but we've done an intermidiat step because moving the `ContentService` now, it means that we have to move all the related consumers